api/models/resource.go

```diff
@@ -27,6 +27,44 @@ type MembersResourceRelation struct {
 	Emails []string `json:"emails"`
 }
 
+// MemberAccess represents that a member has access to a certain resource.
+//  this will get pushed to a device.
+type MemberAccess struct {
+	Email           string
+	ResourceAddress string
+	ResourceName    string
+	Name            string
+	RFID            string
+}
+
+// ACLUpdateRequest is the json object we send to a resource when pushing an update
+type ACLUpdateRequest struct {
+	ACL []string `json:"acl"`
+}
+
+// ACLResponse Response from a resource that is a hash of the ACL that the
+//   resource has stored
+type ACLResponse struct {
+	Hash string `json:"acl"`
```

Comment:
Wouldn't it be better to called it `ACL` instead of Hash or maybe `Id`?

Reply:
probably